web-registry/src/graphql/AllProjects.graphql

```diff
@@ -2,9 +2,11 @@ query AllProjects {
   allProjects {
     nodes {
       id
+      handle
```

Comment:
I used this query it was the closest one to the data needed. Can create a dedicated one if needed.

Reply:
fine to modify 👍